src/keymap.rs

```diff
@@ -28,6 +28,9 @@ pub enum Cmd {
     CapitalizeWord,
     /// clear-screen
     ClearScreen,
+    /// Paste from the clipboard
+    #[cfg(windows)]
```

Comment:
Please, remove `#[cfg(windows]` and use `PasteFromClipboard` [here](https://github.com/kkawakam/rustyline/blob/master/src/keymap.rs#L1070-L1071)

Reply:
> Please, remove `#[cfg(windows]`

You sure about that? I don't want to accidentally break other platforms. Further, the link you gave earlier (https://git.savannah.gnu.org/cgit/readline.git/tree/funmap.c#n123) also had a conditional on windows:

```
#if defined (_WIN32)
  { "paste-from-clipboard", rl_paste_from_clipboard },
#endif
```

> and use PasteFromClipboard here

ok. As an "or", right?

Reply:
I pushed a commit that does this. I'd like to test first on Linux and macOS before merging. I can't do that today - it's already late in my tz :).

Reply:
You are right: if a unix user wants to bind a specific sequence to `PasteFromClipboard`, this would not work because `read_pasted_text` works only in bracketed paste mode...

Reply:
Should I revert https://github.com/kkawakam/rustyline/pull/521/commits/7055c27ff2920f5a2e29cd902b791d2f2fdd3dcf on my branch then? I'm pretty comfortable with the level of testing and "no breakage of other platforms" as of that commit.

Reply:
Yes, please, revert 7055c27.